libs/utils/utils.js

```diff
@@ -638,17 +638,32 @@ const decorateCopyLink = (a, evt) => {
   });
 };
 
+function convertStageLinks(config, anchors) {
```

Comment:
If you export this, you can write proper unit tests for this. I'd even say it's worth passing the as origin for the unit tests, given that this is a complex feature in terms of use-cases it tackles.

We should test
- If each hlx.page|hlx.live|stage.adobe.com|adobe|consumer is transformed and also NOT transformed

Reply:
Added the unit tests 👍